etc/docker/dev/rucio.cfg

```diff
@@ -53,9 +37,9 @@ gss_identity = ddmlab@CERN.CH
 gss_email = ph-adp-ddm-lab@cern.ch
 
 [monitor]
-carbon_server = localhost
+carbon_server = rucio-graphite-int.cern.ch
```

Comment:
won't this spam our graphite server with random people firing up dev containers?

Reply:
Indeed. What is the best way to turn this off entirely for the dev container? Remove the config or point to localhost?

Reply:
leaving it at localhost is fine. maybe later we can even include graphite in the dev container (so people can actually look at the metrics). i think @tbeerman already did that.

Reply:
Thanks @tbeerman I'll do that :-)

Reply:
Ping 😄